usaspending_api/search/v2/views/search.py

```diff
@@ -277,8 +277,8 @@ def post(self, request):
                 m['optional'] = False
 
         json_request = TinyShield(models).block(request.data)
-        fields = json_request.get("fields", None)
-        filters = json_request.get("filters", None)
+        fields = json_request["fields"]
+        filters = json_request.get("filters", {})
```

Comment:
Should this logic also apply to SpendingByAwardCount (line 434)

Reply:
I thought about this, but I didn't want to make those changes without testing. It would also be necessary for SpendingByTransactionCount

Reply:
The SpendingByTransaction count point I agree would be more. I just bring this up considering SpendingByAward and SpendingByAwardCount go hand-in-hand and their numbers should match.

Reply:
I'll add it to SpendingByAwardCount since I have time.
I checked SpendingByTransactionCount and it doesn't support that filter so it is a moot point